qa/L0_backend_python/decoupled/models/decoupled_bls/1/model.py

```diff
@@ -244,13 +286,18 @@ def response_thread(self, response_sender, in_input):
 
         with self.inflight_thread_count_lck:
             self.inflight_thread_count -= 1
+        logger.log("response_thread-Specific Msg!", logger.INFO)
+        logger.log_info("response_thread-Info Msg!")
+        logger.log_warn("response_thread-Warning Msg!")
+        logger.log_error("response_thread-Error Msg!")
 
     def finalize(self):
         """`finalize` is called only once when the model is being unloaded.
         Implementing `finalize` function is OPTIONAL. This function allows
         the model to perform any necessary clean ups before exit.
         """
-        print('Finalize invoked')
```

Comment:
Is there any issues with print or it is just to make the logging consistent?

Reply:
I just wanted to make the logging consistent now that we have the functionality.